pkg/storage/etcd/etcd_helper.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	"k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
```

Comment:
I think the storage layer should be returning storage errors, so this import shouldn't be allowed. The storage errors are translated into api errors at a higher level.